windows.sh

```diff
@@ -33,6 +33,7 @@ fi
 if [ ! -f /usr/bin/ansible ]; then
   echo "Installing pip..."
   sudo apt-get -y install python-pip
+  sudo apt-get -y update
```

Comment:
It seems both ways work the same. I could change it to how you have it before installing pip.